src/styles/styles.css

```diff
@@ -389,6 +391,33 @@ footer {
   font-weight: bolder;
 }
 
+.submit-box {
+  display: flex;
+  justify-content: flex-start;
+  position: relative;
+}
+
+.response-message {
+  text-align: center;
+  display: block;
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 50%;
+}
```

Comment:
- [ ] **The Comment Added Successfully message is not styled correctly. it should be in one line without any line break. also, it would be nice to see some alpha green as a background color and a dark text color. please also make sure to decrease the set timeout function as 1 second is quite too long**

Reply:
Thanks for the observation! I just decreased the set timeout function to 1 second as recommended

However, The comment Adding Successfully message is too long to fit into one line beside the `Comment` button.
An alternative is to make the message appear below the button. That will also make the message wrap for mobile view. Please @shuja-shah  let me know if that is an option you prefer, I think it's best for the message to appear right beside the `Comment` button.

Reply:
Thanks so much @shuja-shah ! I followed your recommendation of adding the response message just after the `Comment` button!
Thanks!!!